### ISSUE TITLE
how to apply SpatialDropout2D() in test time

### ISSUE THREAD

**Issue by andrisecker** [keras-team/keras#9422]
I'm trying to get some kind of uncertainty measure of a CNN's output based on [Gal & Ghahramani 2016](https://arxiv.org/abs/1506.02142). According to an other issue ([#1606](https://github.com/keras-team/keras/issues/1606)) it's easy to leave dropout on during test time, via:
`model.add(Lambda(lambda x: K.dropout(x, level=0.5)))`
I was wondering how to extend this to `SpatialDropout2D()` applied to the convolutional layers, since I get pretty bad results when I start to increase the dropout probability of the convolutional layers...? @fchollet 
Would that make sense from the mathematical point of view? @yaringal

**Comment by fchollet** [comment]
`y = SpatialDropout2D(0.5)(x, training=True)` will apply spatial dropout to `x` at both training and test time.
